### ISSUE TITLE
JavaScript formatting

### ISSUE THREAD

**Issue by CharlesAverill** [CharlesAverill/satyrn#62]
Make [satyrn_backend.js](https://github.com/CharlesAverill/satyrn/blob/master/satyrn_python/static/js/satyrn_backend.js) consistently formatted, as it is currently all over the place.

**Comment by momosetti** [comment]
Hey @CharlesAverill, Could i work on the issue?

**Comment by CharlesAverill** [comment]
@momosetti of course! Just follow the guidelines in the [contributing standards](https://github.com/CharlesAverill/satyrn/blob/master/docs/CONTRIBUTING.md)
